test/parallel/test-quic-internal-endpoint-options.js

```diff
@@ -15,32 +15,24 @@ describe('quic internal endpoint options', { skip: !hasQuic }, async () => {
   } = require('node:assert');
 
   const {
-    Endpoint,
+    QuicEndpoint,
   } = require('internal/quic/quic');
 
   const {
     inspect,
   } = require('util');
 
-  const callbackConfig = {
-    onsession() {},
-    session: {},
-    stream: {},
-  };
-
   it('invalid options', async () => {
     ['a', null, false, NaN].forEach((i) => {
-      throws(() => new Endpoint(callbackConfig, i), {
+      throws(() => new QuicEndpoint(i), {
         code: 'ERR_INVALID_ARG_TYPE',
       });
     });
   });
 
   it('valid options', async () => {
     // Just Works... using all defaults
-    new Endpoint(callbackConfig, {});
-    new Endpoint(callbackConfig);
-    new Endpoint(callbackConfig, undefined);
+    new QuicEndpoint();
```

Comment:
this test no longer tests "all valid options"

Reply:
It never actually did.  I combined the `callbackConfig` and the second options argument. the tests are nowhere near complete and will be a whole thing on their own.